src/lib/localStore.ts

```diff
@@ -0,0 +1,82 @@
+import * as extension from 'extensionizer';
+import log from 'loglevel';
+import { REVERSE_MIGRATIONS } from 'lib/reverseMigrations';
+
+const CURRENT_MIGRATION_VERSION = 0;
+
+export default class ExtensionStore {
+  public isSupported: boolean;
```

Comment:
May be remove this flag already? Keeper can't work without `storage.local`. It would be just useless without it :-)